ethereum_bridge/src/parameters.rs

```diff
@@ -228,6 +281,8 @@ impl EthereumBridgeConfig {
         let eth_start_height = must_read_key(wl_storage, &eth_start_height_key);
 
         Some(Self {
+            // TODO: read this from storage? iterate storage prefix
```

Comment:
exactly

Reply:
I've reworked this to not return the whitelist. the only place where we call `read()` is in the ethereum oracle, which does not need whitelist data